docs/config_file_documentation.txt

```diff
@@ -1,32 +1,29 @@
 This document lists the parameters that can be included in a config file. Required parameters are denoted with an asterisk.
 - *auditOutput: the filename for the audit file that the program will create. Example: portland_2015_mayor_audit_log.txt
-- *visualizerOutput: the filename for the Excel spreadsheet with a summary of the results that the program will create. Example: portland_2015_mayor_visualizer.xlsx
+- *candidates: a list of the candidates who appear in the cast vote records. Each candidate must have a name. If the cast vote records use candidate codes, each candidate must also have a code.
 - contestName: the name of the election. Example: Portland Mayoral Race
+- *cvrFileSources: a list of the cast vote record source files. Each file must have three fields:
+  - *filePath: the location of the file. Example: test/2015-portland-mayor-cvr.xlsx
+  - *firstVoteColumnIndex: the index of the column that contains each ballot’s top-ranked candidate. Example: 3
+  - precinctColumnIndex: the index of the column that contains each ballot's precinct name. Example: 2
+  - *provider: the voting machine format of the file. Example: ES&S
+- date: the date of the election. Example: 2015-11-03
 - jurisdiction: the jurisdiction in which this election is being contested. Example: Portland, ME
 - office: the office being contested. Example: Mayor
-- date: the date of the election. Example: 2015-11-03
-- *maxRankingsAllowed: the maximum number of candidates that a ballot is allowed to rank. Example: 15
+- precinctOutputDirectory: the path to where the precinct spreadsheets should be saved. Example: portland_2015_precinct_output (defaults to the current directory)
```

Comment:
I think that we should have a general `outputDirectory` field (i.e. where the log and visualizer files go) in each config file instead of a special one for precincts only. Been wanting to make this happen for a little while now. Think we can work it into this branch before the merge?

Reply:
This is fine, and I'm thinking maybe we should go one step further and stop letting the user specify output file names entirely to simplify things.  They just enter the output folder, and we generate names for the outputs like 
<guid>_visualizer.xls,  <guid>_audit.txt and so on (new guid for every tabulation)